sw/device/silicon_creator/lib/base/chip.h

```diff
@@ -43,16 +43,15 @@
 #define CHIP_BOOT_SVC_MSG_HEADER_SIZE 44
 
 /**
- * Maximum payload size for a boot services message.
+ * Maximum size of a boot services message.
  */
-#define CHIP_BOOT_SVC_MSG_PAYLOAD_SIZE_MAX 256
+#define CHIP_BOOT_SVC_MSG_SIZE_MAX 48
```

Comment:
As of now, this is the maximum size of any currently existing boot services message.  Is there a theoretical maximum that no boot services message can ever exceed? (either now or in the future).

Reply:
Yeah, this is the maximum size of any currently existing boot services message. This is primarily used to determine the random payload of the empty message that can be used to clear any existing boot services message.

Reply:
As for the theoretical maximum, it's probably roughly the half of the retention SRAM but we will probably add new entries for the silicon creator so it will be less eventually.

Reply:
It would be good to have a little more context for this constant and why it's necessary. It's not clear to me without dissecting the `BOOT_SVC_SIZE_*` macros below.

Reply:
I thought the comment was clear enough. I'll see if I can come up with a better explanation but this file is basically a collection of global constants.